services/catalog/tests/unit/conftest.py

```diff
@@ -145,7 +147,10 @@ async def app(
 
     # create instance
     assert app_environment
-    app_under_test = create_app()
+    tracing_data = TracingData.create(
+        service_name=APP_NAME, tracing_settings=None  # disable tracing in tests
+    )
```

Comment:
if you want to make it more explicit, make a method `created_disabled` where you set tracking_settings to None this way it's more readable and you do not require comments